src/pages/EditSplitBillPage.js

```diff
@@ -0,0 +1,161 @@
+import Navigation from '../libs/Navigation/Navigation';
+import FullPageNotFoundView from '../components/BlockingViews/FullPageNotFoundView';
+import EditRequestDescriptionPage from './EditRequestDescriptionPage';
+import EditRequestMerchantPage from './EditRequestMerchantPage';
+import EditRequestCreatedPage from './EditRequestCreatedPage';
+import EditRequestAmountPage from './EditRequestAmountPage';
+
+const propTypes = {
+    /** Route from navigation */
+    route: PropTypes.shape({
+        /** Params from the route */
+        params: PropTypes.shape({
+            /** The transaction field we are editing */
+            field: PropTypes.string,
+
+            /** The chat reportID of the split */
+            reportID: PropTypes.string,
+
+            /** reportActionID of the split action */
+            reportActionID: PropTypes.string,
+        }),
+    }).isRequired,
+
+    /** The current transaction */
+    transaction: transactionPropTypes.isRequired,
+
+    /** The draft transaction that holds data to be persisted on the current transaction */
+    draftTransaction: PropTypes.shape(transactionPropTypes),
+};
+
+const defaultProps = {
+    draftTransaction: {},
+};
+
+function EditSplitBillPage({route, transaction, draftTransaction}) {
+    const fieldToEdit = lodashGet(route, ['params', 'field'], '');
+    const reportID = lodashGet(route, ['params', 'reportID'], '');
+    const reportActionID = lodashGet(route, ['params', 'reportActionID'], '');
+
+    const {
+        amount: transactionAmount,
+        currency: transactionCurrency,
+        comment: transactionDescription,
+        merchant: transactionMerchant,
+        created: transactionCreated,
+    } = draftTransaction ? ReportUtils.getTransactionDetails(draftTransaction) : ReportUtils.getTransactionDetails(transaction);
+
+    const defaultCurrency = lodashGet(route, 'params.currency', '') || transactionCurrency;
+
+    function navigateBackToSplitDetails() {
+        Navigation.navigate(ROUTES.SPLIT_BILL_DETAILS.getRoute(reportID, reportActionID));
+    }
+
+    function setDraftSplitTransaction(transactionChanges) {
+        IOU.setDraftSplitTransaction(transaction.transactionID, transactionChanges);
+        navigateBackToSplitDetails();
+    }
+
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.DESCRIPTION) {
+        return (
+            <EditRequestDescriptionPage
+                defaultDescription={transactionDescription}
+                onSubmit={(transactionChanges) => {
+                    setDraftSplitTransaction({
+                        comment: transactionChanges.comment.trim(),
+                    });
+                }}
+            />
+        );
+    }
+
+    if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.DATE) {
+        return (
+            <EditRequestCreatedPage
+                defaultCreated={transactionCreated}
+                defaultAmount={transactionAmount}
+                reportID={reportID}
```

Comment:
FYI, this component doesn't use either of these props. I'm going to remove them in my PR and I'll ping you on it.